### PR TITLE
docs(all): fix typos

### PR DESCRIPTION
**Pull request by omahs**
Fix: typos

### CODE REVIEW

**Review by qdm12 — APPROVED** [review]
Nice, thanks 👍 

Maybe you could even setup a misspell CI check like https://github.com/w3f/polkadot-spec/pull/509/commits/f91842159e123a523afe62e8d4e934d4f4c52bf0 but really an optional, additional step 😉

**Review by EclesioMeloJunior — APPROVED** [review]

**Comment by omahs** [comment]
@qdm12 I just added the same misspelling check, hope it's ok (not a dev 😀)

File: .github/workflows/misspell.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  misspell:
+    name: Misspelling check
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3.2.0
+      - uses: reviewdog/action-misspell@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          locale: "US"
```
Comment (EclesioMeloJunior) [inline .github/workflows/misspell.yml:18]: This should fix the CI failures
```suggestion
          locale: "UK"
```

**Review by EclesioMeloJunior — CHANGES_REQUESTED** [review]

File: .github/workflows/misspell.yml (hunk 1 above)
Comment (EclesioMeloJunior) [reply .github/workflows/misspell.yml]: @omahs, this change was not enough to solve the `Misspelling Check` CI problems 😅, you will need to resolve each complaint of the checker, most of them are misspelled:

```
Error: [misspell] reported by reviewdog 🐶
"decentralized" is a misspelling of "decentralised"

Error: [misspell] reported by reviewdog 🐶
"amortizes" is a misspelling of "amortises"
...
```

you can find all here -> https://github.com/ChainSafe/gossamer/actions/runs/4068018700/jobs/7021954302

Comment (omahs) [reply .github/workflows/misspell.yml]: @EclesioMeloJunior almost all of them are correct US spellings, I changed "package" which was misspelled. Not sure if this workflow is useful if it creates issues like that.

Comment (omahs) [reply .github/workflows/misspell.yml]: @EclesioMeloJunior Perfect, just did that!

**Pull request merged**